ci/prow/make_config.go

```diff
@@ -420,6 +422,32 @@ tide:
     [[indent_section 4 "volumes" .Base.Volumes]]
 `
 
+  // flakyTestReporterPeriodicJob is the template for the flaky tests reporter job.
```

Comment:
Can you please create the job from the standard periodic job template? Properly setting the command and args should be enough.

Reply:
using standard periodic job template, please take a look

Reply:
Updated with laster master change, it's cleaner now. @adrcunha  please review again